lib/engine_manager.rb

```diff
@@ -1,17 +1,16 @@
-#
 # Keeper of this app's loaded engines
 class EngineManager
 
-  @@loaded_engines={}
+  @@loaded_engines = {}
 
   def self.engine_loaded?(engine_name)
-    normalized=engine_name.to_s.camelize.to_sym
+    class_name = "#{engine_name.to_s.camelize}::Engine"
```

Comment:
I think going from `start_with?` to a more precise check makes sense, but what if we cleaned up the logic some?

```ruby
# note sure if it's worth memoizing or not
def self.loaded_engines
  Rails.application.railties.engines.map { |e| e.class.name }.to_set
end

def self.engine_loaded?(engine_name)
  class_name = "#{engine_name.to_s.camelize}::Engine"
  loaded_engines.include?(class_name)
end
```